BlackNectar/BlackNectar/SearchStores.swift

```diff
@@ -15,9 +15,21 @@ class SearchStores {
     typealias Callback = ([StoresInfo]) -> ()
     
     // API Call
-    static func searchForStoresLocations(near point: CLLocationCoordinate2D, callback: @escaping Callback) {
+    static func searchForStoresLocations(near point: CLLocationCoordinate2D, with radius: Double, callback: @escaping Callback) {
```

Comment:
Better to name the paramters as `withRadius radius: Double` though.
Because we may have other options in the request, like `onlyOpenNow` etc. `with` by itself is not specific enough.